examples/platforms/da15000/README.md

```diff
@@ -0,0 +1,102 @@
+# Dialog OpenThread DA15000 Example
+
+This directory contains example platform drivers for the [Dialog Semiconductor DA15000][da15000]
+
+[da15000]: https://support.dialog-semiconductor.com/connectivity/product/openthread-sandbox
+
+## Build Examples (How to build and flash):
+
+```bash
+$ cd <path-to-openthread>
+$ ./bootstrap
+$ make -f examples/Makefile-da15000 clean
+$ make -f examples/Makefile-da15000
+```
+
+Execute script to prepare image for flashing:
+
+```bash
+$ cd ./third_party/dialog/DialogTools
+$ ./imgprep.sh
+```
+
+Flash Binaries:
+
+1. Open another terminal window in order to start gdb server. Write the command (Linux):
+   ```bash
+   $ JLinkGDBServer -ir -if swd -speed auto -device Cortex-M0 -logtofile on -localhostonly
```

Comment:
This command is not included in the Dialog SDK and is not a standard utility, could you please provide a link in this README as to where to get this utility.